core/src/main/scala/com/databricks/labs/remorph/coverage/QueryRunner.scala

```diff
@@ -17,13 +17,16 @@ abstract class BaseQueryRunner(transpiler: Transpiler) extends QueryRunner {
       exampleQuery: ExampleQuery,
       output: String,
       error: Option[RemorphError] = None): ReportEntryReport = {
-    val expected = exampleQuery.expectedTranslation.getOrElse("")
+    val expected = exampleQuery.expectedTranslation
     val parsed = if (error.isEmpty) 1 else 0
-    if (exampleQuery.expectedTranslation.map(format).exists(_ != format(output))) {
+    val formattedOutput = if (exampleQuery.shouldFormat) format(output) else output
+    val formattedExpected = expected.map(e => if (exampleQuery.shouldFormat) format(e) else e)
```

Comment:
I think this can be tidied up a bit:
```suggestion
    val formattedOutput = if (exampleQuery.shouldFormat) format(output) else output
    val formattedExpected = if (exampleQuery.shouldFormat) expected.map(format))
```
There's a variant also possible, but I don't think it's really an improvement:
```suggestion
    val (formattedOutput, formattedExpected) = {
      if (exampleQuery.shouldFormat) {
        (format(output), expected.map(format))
      } else {
        (output, expected)
      }
    }
```